### DISCUSSION TITLE
Load assets locally

### DISCUSSION THREAD

**Discussion by emabusi2** [Laravel-Backpack/community-forum#980] [Q&A (Help)]
How can I make the assets to load without using the unpkg.com cdn and take from the asset itself

**Comment by karandatwani92** [comment]
Hey @emabusi2 

Backpack uses [basset](https://github.com/Laravel-Backpack/basset), which caches CDN files and servers from the local assets.

Setting your `APP_ENV` in production will enable this behavior. If you want the same in local/dev environment, add `BASSET_DEV_MODE=false` to the env.
